### PR TITLE
Add bio for Kristy

### LINKED ISSUE #18: Add bio for Kristy
- [ ] Create a Branch
- [ ] Add a bio file for mlmallo
- [ ] Open a pull request
- [ ] Get and receive feedback
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by Kristy46549871321**
This pull request fixes #18 by creating a new bio file for Kristy.

### CODE REVIEW

File: Kristy.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#Kristy
+
+* Name: Kristy
```
Comment (beltramie) [inline Kristy.md:3]: Correct!

**Comment by beltramie** [comment]
If you have a pet, can you add that pets name?

**Comment by Kristy46549871321** [comment]
@beltramie What do you think now? Is it good?

**Comment by beltramie** [comment]
Looking good :100:

**Comment by beltramie** [comment]
oops I merged your branch instead of mine, feel free to merge my pull request and delete the branch

**Pull request merged**